TODO.md

```diff
@@ -29,3 +29,6 @@ Items to build, research, test or evaluate in creating the services outlined abo
 6. Metrics to collect on the server portion of the processing (prometheus metrics)
 
 7. Finding a golang MRT reader is also a required work item,
+
+8. Integrate RouteViews data with MLab data in BigQuery?
```

Comment:
this would be awesome.

also I wiped out your changes somehow?
I suspect this is because: "Chris is bad at git" :(